package.json

```diff
@@ -10,6 +8,7 @@
     "gulp-cssnano": "^2.1.0",
     "gulp-sass": "^2.1.1",
     "gulp-sourcemaps": "^1.6.0",
+    "mocha": "^2.3.4",
```

Comment:
and what I mean by that is, we need to add mocha to `package.json` in case this PR doesn‘t get merged.